### ISSUE TITLE
What’s the difference between this project and WXinlong/SOLO?

### ISSUE THREAD

**Issue by longbowking** [Fei-dong/SOLO_SORT#1]
is this repo a clone of https://github.com/WXinlong/SOLO?

**Comment by fdong0598** [comment]
> this repo is just a clone of https://github.com/WXinlong/SOLO
No, I use this solo as the location part to add tracking based on it.

**Comment by fdong0598** [comment]
> > No, I use this solo as the location part to add tracking based on it.

The details are as follows demo/webcam_demo.py

**Comment by longbowking** [comment]
thanks for your share
